lib/data/network/contact/tom_contact_api.dart

```diff
@@ -10,9 +10,12 @@ import 'package:fluffychat/domain/model/contact/lookup_mxid_request.dart';
 import 'package:fluffychat/domain/model/contact/lookup_mxid_response.dart';
 
 class TomContactAPI {
-  final DioClient _client =
+  final DioClient _identityClient =
       getIt.get<DioClient>(instanceName: NetworkDI.identityDioClientName);
 
+  final DioClient _tomClient =
+      getIt.get<DioClient>(instanceName: NetworkDI.tomDioClientName);
```

Comment:
we should have to separate it. `identity` server in the future will be a federation identify, which can share the identity with connected homeservers. `tom` is only for an organization.

Reply:
done